rpc/handlers.go

```diff
@@ -24,6 +29,10 @@ var (
 	ErrPageSizeTooBig           = &jsonrpc.Error{Code: 31, Message: "Requested page size is too big"}
 	ErrTooManyKeysInFilter      = &jsonrpc.Error{Code: 34, Message: "Too many keys provided in a filter"}
 	ErrInternal                 = &jsonrpc.Error{Code: jsonrpc.InternalError, Message: "Internal error"}
+	ErrCodeInvalidParams        = -32602
+	ErrCodeOutOfRangeFee        = -32603
+	ErrCodeOutOfRangeContract   = -32604
+	ErrCodeUnknownError         = -32605
```

Comment:
@rianhughes codes -32602 and -32603 already defined in `jsonrpc` package. Also looks like -32604 and -32605 are not part of [predefined errors from the spec](https://www.jsonrpc.org/specification#:~:text=NOT%20be%20included.-,5.1%20Error%20object,-When%20a%20rpc).

Reply:
Nice catch, I've updated the code to use predefined jsonrpc codes.